Warnings logged by the build of jena-integration-tests about geosparql

### Version

5.5.0 dev

### Question

The build is showing some wanrings from geosparql.

@Aklakan --
Question: are these expected? - if so, I'll configure log4j to suppress these warnings.

```
[INFO] [INFO] Running org.apache.jena.geosparql.TestGeoAssembler
[ERROR] 12:12:47 WARN  org.apache.jena.geosparql.assembler.GeoAssembler :: file:///home/afs/ASF/Release/jena/jena-integration-tests/src/test/files/GeoAssembler/geo-config-mem.ttl#geo_ds: No spatial index file location is specified via http://jena.apache.org/geosparql#spatialIndexFile. Spatial index will not be persisted.
[ERROR] 12:12:47 WARN  org.apache.jena.geosparql.assembler.GeoAssembler :: file:///home/afs/ASF/Release/jena/jena-integration-tests/src/test/files/GeoAssembler/geo-config-mem.ttl#geo_ds: No preferred SRS is configured via http://jena.apache.org/geosparql#srsUri
[ERROR] 12:12:47 WARN  org.apache.jena.geosparql.assembler.GeoAssembler :: file:///home/afs/ASF/Release/jena/jena-integration-tests/src/test/files/GeoAssembler/fuseki-geo-text-tdb2.ttl#testGeoDS: No spatial index file location is specified via http://jena.apache.org/geosparql#spatialIndexFile. Spatial index will not be persisted.
[ERROR] 12:12:47 WARN  org.apache.jena.geosparql.assembler.GeoAssembler :: file:///home/afs/ASF/Release/jena/jena-integration-tests/src/test/files/GeoAssembler/fuseki-geo-text-tdb2.ttl#testGeoDS: No preferred SRS is configured via http://jena.apache.org/geosparql#srsUri
[ERROR] 12:12:47 WARN  org.apache.jena.geosparql.assembler.GeoAssembler :: Dataset is empty. Constructing an empty spatial index that needs to be updated once data is added.
[ERROR] 12:12:47 WARN  org.apache.jena.geosparql.configuration.GeoSPARQLOperations :: SRS scan completed. No SRS usage found. Falling back to default SRS: http://www.opengis.net/def/crs/OGC/1.3/CRS84
[ERROR] 12:12:47 WARN  org.apache.jena.geosparql.assembler.GeoAssembler :: file:///home/afs/ASF/Release/jena/jena-integration-tests/src/test/files/GeoAssembler/geo-config.ttl#geo_ds: No preferred SRS is configured via http://jena.apache.org/geosparql#srsUri
[ERROR] 12:12:48 WARN  org.apache.jena.geosparql.assembler.GeoAssembler :: file:///home/afs/ASF/Release/jena/jena-integration-tests/src/test/files/GeoAssembler/geo-config-ex.ttl#geo_ds: No preferred SRS is configured via http://jena.apache.org/geosparql#srsUri
[ERROR] 12:12:48 WARN  org.apache.jena.geosparql.assembler.GeoAssembler :: file:///home/afs/ASF/Release/jena/jena-integration-tests/src/test/files/GeoAssembler/fuseki-text-geo-tdb2.ttl#testGeoDS: No spatial index file location is specified via http://jena.apache.org/geosparql#spatialIndexFile. Spatial index will not be persisted.
[ERROR] 12:12:48 WARN  org.apache.jena.geosparql.assembler.GeoAssembler :: file:///home/afs/ASF/Release/jena/jena-integration-tests/src/test/files/GeoAssembler/fuseki-text-geo-tdb2.ttl#testGeoDS: No preferred SRS is configured via http://jena.apache.org/geosparql#srsUri
[ERROR] 12:12:48 WARN  org.apache.jena.geosparql.assembler.GeoAssembler :: Dataset is empty. Constructing an empty spatial index that needs to be updated once data is added.
[ERROR] 12:12:48 WARN  org.apache.jena.geosparql.configuration.GeoSPARQLOperations :: SRS scan completed. No SRS usage found. Falling back to default SRS: http://www.opengis.net/def/crs/OGC/1.3/CRS84
[ERROR] 12:12:48 WARN  org.apache.jena.geosparql.assembler.GeoAssembler :: file:///home/afs/ASF/Release/jena/jena-integration-tests/src/test/files/GeoAssembler/geo-assembler.ttl#geo_ds: No spatial index file location is specified via http://jena.apache.org/geosparql#spatialIndexFile. Spatial index will not be persisted.
[ERROR] 12:12:48 WARN  org.apache.jena.geosparql.assembler.GeoAssembler :: file:///home/afs/ASF/Release/jena/jena-integration-tests/src/test/files/GeoAssembler/geo-assembler.ttl#geo_ds: No preferred SRS is configured via http://jena.apache.org/geosparql#srsUri
[ERROR] 12:12:48 WARN  org.apache.jena.geosparql.assembler.GeoAssembler :: Dataset is empty. Constructing an empty spatial index that needs to be updated once data is added.
[ERROR] 12:12:48 WARN  org.apache.jena.geosparql.configuration.GeoSPARQLOperations :: SRS scan completed. No SRS usage found. Falling back to default SRS: http://www.opengis.net/def/crs/OGC/1.3/CRS84
```

--- a/jena-integration-tests/src/test/resources/log4j2-test.properties
+++ b/jena-integration-tests/src/test/resources/log4j2-test.properties
@@ -49,8 +49,20 @@ logger.fuseki-auto.level = ERROR
 logger.jetty.name  = org.eclipse.jetty
 logger.jetty.level = WARN
 
+## Various warnings
+
 logger.geosparql-registry.name = org.apache.jena.geosparql.implementation.registry.SRSRegistry
 logger.geosparql-registry.level = ERROR
+
+logger.geosparql-assembler.name = org.apache.jena.geosparql.assembler.GeoAssembler
+logger.geosparql-assembler.level = ERROR
+
+logger.geosparql-operations.name = org.apache.jena.geosparql.configuration.GeoSPARQLOperations
+logger.geosparql-operations.level = ERROR
+
+logger.lucene-vectorization.name = org.apache.lucene.internal.vectorization.VectorizationProvider
+logger.lucene-vectorization.level = ERROR
+
 
 # This goes out in NCSA format
 appender.plain.type = Console

